judge/judge.py

```diff
@@ -163,6 +175,9 @@ def _judge_impl(judge_request: JudgeRequest, process_id: int) -> Union[Verdict,
                 else:
                     test_case_result.verdict = Verdict.WA
                     test_case_result.score = 0.0
+                    
+        if submission.in_contest and test_case_result.verdict != Verdict.AC:

```

Comment:
is it better to use `test_case_result.score == 0.0`? since partial score don't want skip

Reply:
I agree.